src/material/checkbox/_checkbox-theme.scss

```diff
@@ -31,9 +33,13 @@
 /// @param {ArgList} Additional optional arguments (only supported for M3 themes):
 ///   $color-variant: The color variant to use for the checkbox: primary, secondary, tertiary, or
 ///     error (If not specified, default primary color will be used).
-@mixin color($theme, $options...) {
+@mixin color($theme, $custom-tokens: (), $options...) {
```

Comment:
IMO we shouldn't add tokens to the existing mixins. We should just one extra mixin like `customize` (name to be debated) where users pass in a flat map of tokens.